### PR TITLE
Added a new quote.

### PR DESCRIPTION
**Pull request by triach-rold**
### Description

Just gave the credits to the original author of the quote. This is a common misattribution. [Source](https://www.goodreads.com/quotes/467-life-is-what-happens-to-us-while-we-are-making).

### CODE REVIEW

**Comment by triach-rold** [comment]
Added a new quote from Friedrich Nietzsche.

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38615,6 +38615,12 @@
       "source": "Leto Atreides II, God Emperor of Dune",
       "length": 107,
       "id": 7058
+    },
+    {
+      "text": "He who has a why to live for can bear almost any how.",
+      "source": "Friedrich Nietzsche",
+      "length": 53,
+      "id": 7059
```
[the remark this answers is not in the document]
Comment (triach-rold) [reply frontend/static/quotes/english.json]: You're right. I've fixed the punctuation @Miodec !

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8983579016

**Comment by Miodec** [comment]
Please fix incorrect quote length and PR title

**Comment by triach-rold** [comment]
> Please fix incorrect quote length and PR title

Quote length and PR title has been fixed, as requested @Miodec

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38615,6 +38615,12 @@
       "source": "Leto Atreides II, God Emperor of Dune",
       "length": 107,
       "id": 7058
+    },
+    {
+      "text": "He who has a 'why' to live for can bear almost any 'how'.",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38620]: Use double quotes (dont forget to escape `\"`) instead of singles.

**Review by Miodec — CHANGES_REQUESTED** [review]
Please update pr title aswell.

File: frontend/static/quotes/english.json (hunk 2 above)
Comment (triach-rold) [reply frontend/static/quotes/english.json:38620]: Done.

Comment (triach-rold) [reply frontend/static/quotes/english.json:38620]: @Miodec I believe the requested changes have been made ?

**Comment by Miodec** [comment]
Please resolve conflicts

**Comment by Warrobot10** [comment]
Please change the pr title to something like impr(quotes): added a new quote(triach-rold)
We do this so we can easily understand what area the change focuses on, and then allows the release system to list the changes in the release notes.

**Pull request closed without merging**